.github/workflows/optimize-release-optimize-c8-only.yml

```diff
@@ -190,11 +187,24 @@ jobs:
             -Darguments="-Dmaven.deploy.skip=${IS_DRY_RUN} -DskipTests -DskipNexusStagingDeployMojo=${IS_DRY_RUN} -B --fail-at-end -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -f pom.xml"
 
       - name: Remove tag for RC release
-        if: env.IS_RC == 'true'
+        if: env.IS_RC == 'true' || failure()
         run: |
           git tag -d ${TAG}
           git push origin :refs/tags/${TAG}
 
+      - name: Create a GitHub release
+        if: ${{ env.IS_RC == 'false' && env.DRY_RUN == 'false' }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: octokit/request-action@v2.3.1
+        with:
+          route: POST /repos/camunda/zeebe/releases
```

Comment:
```suggestion
          route: POST /repos/camunda/camunda/releases
```
?

Reply:
this repository is `camunda/camunda` not `camunda/zeebe` anymore

Reply:
example, so my suggestion is correct?